src/pages/Cart/component/CartBox.js

```diff
@@ -1,22 +1,115 @@
+import { useState } from 'react';
 import './CartBox.scss';
-const CartBox = () => {
+
+const CartBox = ({ product, setProductList, id, setCartList }) => {
+  const { image_url, name, quantity, product_id } = product;
+  const [count, setCount] = useState(quantity);
+  let totalPrice = Number(product.price * product.quantity).toLocaleString(
+    'en'
+  );
+  const deleteCart = id => {
+    setCartList(prev => prev.filter(product => product.product_id !== id));
+  };
+  const handleCountMinus = () => {
```

Comment:
```suggestion
  const [count, setCount] = useState(quantity);
  
  let totalPrice = Number(product.price * product.quantity).toLocaleString(
    'en'
  );
  
  const deleteCart = id => {
    setCartList(prev => prev.filter(product => product.product_id !== id));
  };
  
  const handleCountMinus = () => {
```

코드의 가독성을 높이는 것도 코드를 작성할 때 중요한 부분입니다.
컨벤션은 아니지만 저 같은 경우 hook은 띄어쓰기를 사용하지 않지만
함수끼리는 한칸씩 띄워서 가독성을 높여주는 방식으로 적용해 줍니다!

Reply:
추가로 부모로 부터 받은 값은 state로 만들지 않으셔도 됩니다!